lib/report_formatter/timeline.rb

```diff
@@ -120,11 +120,11 @@ def tl_event(row, col)
                         rec[:container_node_name]
                       end
           end
-          e_title ||= ems ? ems.name : "No VM, Host, or MS"
         else
           e_title = rec[:name] ? rec[:name] : row[mri.col_order.first].to_s
         end
       end
+      e_title ||= ems ? ems.name : "No VM, Host, or MS"
```

Comment:
I think this move needs some explanation :)

Reply:
Line 179 assumes there's a string in `e_title`:
https://github.com/ManageIQ/manageiq-ui-classic/pull/4973/files#diff-5a3e558e1c263ef2003287586a2e6f1bR179

For some strange reason the fallback for `e_title` to use a value from `ems` what only active when there was a `@record` -- that actually does not make much sense.

So I am moving the fallback out of the conditions -- a change that does not really change anything in the practical situations.

That makes the code work even if the `rec` is `nil` and that is the case when you run a test w/o writing the new records in the database.